checkpoint/runner.go

```diff
@@ -23,59 +22,21 @@ const (
 	dirPerm       = 0o700
 )
 
-type Runner struct {
-	fs     afero.Fs
-	db     *sql.Database
-	logger log.Log
-	dir    string
-}
-
-type Opt func(*Runner)
-
-func WithDataDir(dir string) Opt {
-	return func(r *Runner) {
-		r.dir = dir
-	}
-}
-
-func WithLogger(lg log.Log) Opt {
-	return func(r *Runner) {
-		r.logger = lg
-	}
-}
-
-func WithFilesystem(fs afero.Fs) Opt {
-	return func(r *Runner) {
-		r.fs = fs
-	}
-}
-
-func NewRunner(db *sql.Database, opts ...Opt) *Runner {
-	r := &Runner{
-		fs:     afero.NewOsFs(),
-		db:     db,
-		logger: log.NewNop(),
-	}
-	for _, opt := range opts {
-		opt(r)
-	}
-	return r
-}
-
-func checkpointDB(ctx context.Context, db *sql.Database, snapshot, restore types.LayerID) (*Checkpoint, error) {
+func checkpointDB(ctx context.Context, db *sql.Database, snapshot types.LayerID) (*Checkpoint, error) {
 	checkpoint := &Checkpoint{
 		Version: SchemaVersion,
 		Data: InnerData{
-			CheckpointId: fmt.Sprintf("snapshot-%d-restore-%d", snapshot, restore),
-			Restore:      restore.Uint32(),
+			CheckpointId: fmt.Sprintf("snapshot-%d", snapshot),
 		},
 	}
 
 	tx, err := db.Tx(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("create db tx: %s", err)
 	}
-	defer tx.Release()
```

Comment:
why?

Reply:
it's replaced by the line below. (my IDE complained about un-captured error

Reply:
i noticed. i probably commented on wrong line. just seemed strange. maybe setting for ide have to be fixed, linter in go-spacemesh doesn't complain on this